### PR TITLE
Load dateformat for InsightView component

### PR DESCRIPTION
**Pull request by lanhq**
JIRA: SD-1179

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [x] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [x] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1387/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2978/ : FAILURE in 2m 41s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1389/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2980/ : FAILURE in 1m 27s

**Comment by lanhq** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1407/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/459/ : SUCCESS in 12m 57s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1406/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2998/ : SUCCESS in 13m 53s

File: libs/sdk-ui-ext/src/insightView/tests/dataLoaders.test.ts [hunk 1]
```diff
@@ -89,48 +89,58 @@ describe("InsightViewDataLoader", () => {
...
                 settings: () => ({
-                    getSettingsForCurrentUser: noop as any,
-                    getSettings: getSettings,
+                    getSettingsForCurrentUser: getSettingsForCurrentUser,
+                    getSettings: noop as any,
                 }),
             }),
         });
 
         it("should cache workspaceSettings calls", async () => {
```
Comment (anvuong119) [inline libs/sdk-ui-ext/src/insightView/tests/dataLoaders.test.ts:109]: should we change the title to "should cache userWorkspaceSettings calls"?

File: libs/sdk-ui-ext/src/insightView/tests/dataLoaders.test.ts [hunk 2]
```diff
@@ -89,48 +89,58 @@ describe("InsightViewDataLoader", () => {
     });
 
     describe("workspaceSettings calls", () => {
```
Comment (anvuong119) [inline libs/sdk-ui-ext/src/insightView/tests/dataLoaders.test.ts:91]: should we change the title to "userWorkspaceSettings calls"?

File: libs/sdk-ui-ext/src/insightView/tests/dataLoaders.test.ts [hunk 3]
```diff
@@ -89,48 +89,58 @@ describe("InsightViewDataLoader", () => {
...
+            const second = loader.getUserWorkspaceSettings(backend);
 
             const [firstResult, secondResult] = await Promise.all([first, second]);
 
             expect(secondResult).toBe(firstResult);
-            expect(getSettings).toHaveBeenCalledTimes(1);
+            expect(getSettingsForCurrentUser).toHaveBeenCalledTimes(1);
         });
 
         it("should not cache workspaceSettings errors", async () => {
```
Comment (anvuong119) [inline libs/sdk-ui-ext/src/insightView/tests/dataLoaders.test.ts:125]: should we change the title to "should not cache userWorkspaceSettings errors"?

**Comment by anvuong119** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1408/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2999/ : SUCCESS in 13m 50s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/482/ : SUCCESS in 22s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/741/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3000/ : SUCCESS in 13m 51s

**Pull request merged**